great_expectations/dataset/dataset.py

```diff
@@ -1252,8 +1255,6 @@ def expect_column_values_to_be_between(self,
                 If True, then catch exceptions and include them as part of the result object. \
                 For more detail, see :ref:`catch_exceptions`.
             meta (dict or None): \
-                A JSON-serializable dictionary (nesting allowed) that will be included in the output without \
```

Comment:
Probably you removed this accidentaly

Reply:
Yep, you are completely right, thank you for reviewing!